code/parachain/frame/cosmwasm/src/ibc.rs

```diff
@@ -66,6 +72,26 @@ impl<T: IbcHandler<AccountIdOf<C>>, C: Config> IbcHandlerExtended<C> for T {
 	}
 }
 
+pub struct ChannelOpenCall;
```

Comment:
why don't we use this? https://github.com/ComposableFi/cosmwasm-vm/blob/main/vm/src/executor.rs#L528

Reply:
it seems lacks some traits and fails to compile

Reply:
if fix is needed, will be done in separate PR because will require fixes to other repos